posthog/hogql_queries/insights/funnels/funnel_trends_udf.py

```diff
@@ -195,7 +196,9 @@ def get_query(self) -> ast.SelectQuery:
             """,
                 {"fill_query": fill_query, "inner_select": inner_select},
             )
-        return cast(ast.SelectQuery, s)
+        s = cast(ast.SelectQuery, s)
+        s.settings = HogQLQuerySettings(join_algorithm=JOIN_ALGOS)
```

Comment:
might be safer to first check emptiness and then override `s.settings`, cause if in future `parse_select` starts setting query settings as well, those will be lost